saleor/tests/e2e/vouchers/test_export_voucher_codes.py

```diff
@@ -0,0 +1,496 @@
+import os
+import shutil
+import uuid
+
+import pytest
+
+from ..shop.utils.preparing_shop import prepare_shop
+from ..utils import assign_permissions
+from .utils import (
+    create_voucher,
+    create_voucher_channel_listing,
+    export_voucher_codes,
+    raw_export_voucher_codes,
+)
+
+
+@pytest.fixture
+def media_root_with_clean(tmpdir, settings):
+    root = str(tmpdir.mkdir("media"))
+    settings.MEDIA_ROOT = root
+
+    yield root
+
+    if os.path.exists(root):
+        shutil.rmtree(root)
+
+
+def create_vouchers_with_multiple_codes(e2e_staff_api_client, channel_id):
+    voucher_ids = []
+    voucher_code_ids = []
+
+    for i in range(5):
+        voucher_code = f"voucher_code_{i + 1}_{uuid.uuid4()}"
+
+        input_data = {
+            "addCodes": [voucher_code],
+            "discountValueType": "PERCENTAGE",
+            "type": "ENTIRE_ORDER",
+        }
+        voucher_data = create_voucher(e2e_staff_api_client, input_data)
+
+        voucher_id = voucher_data["id"]
+        for i in range(len(voucher_data["codes"]["edges"])):
+            voucher_code_id = voucher_data["codes"]["edges"][i]["node"]["id"]
+            voucher_code_ids.append(voucher_code_id)
+
+        channel_listing = [
+            {
+                "channelId": channel_id,
+                "discountValue": i + 1,
+            },
+        ]
+        create_voucher_channel_listing(
+            e2e_staff_api_client,
+            voucher_id,
+            channel_listing,
+        )
+
+        voucher_ids.append(voucher_id)
+
+    return voucher_ids, voucher_code_ids
+
+
+@pytest.mark.e2e
+@pytest.mark.usefixtures("media_root_with_clean")
+@pytest.mark.parametrize(
+    ("file_type", "voucher_id_array_index", "expected_status"),
+    [
+        ("CSV", 0, "SUCCESS"),
+        ("XLSX", 2, "SUCCESS"),
+    ],
+)
+def test_export_valid_voucher_ids_CORE_0925(
+    e2e_staff_api_client,
+    permission_manage_channels,
+    permission_manage_shipping,
+    permission_manage_products,
+    permission_manage_discounts,
+    permission_manage_checkouts,
+    file_type,
+    voucher_id_array_index,
+    expected_status,
+):
```

Comment:
Maybe we could use exisitng `media_root` fixture? 
```suggestion
@pytest.mark.e2e
@pytest.mark.parametrize(
    ("file_type", "voucher_id_array_index", "expected_status"),
    [
        ("CSV", 0, "SUCCESS"),
        ("XLSX", 2, "SUCCESS"),
    ],
)
def test_export_valid_voucher_ids_CORE_0925(
    e2e_staff_api_client,
    permission_manage_channels,
    permission_manage_shipping,
    permission_manage_products,
    permission_manage_discounts,
    permission_manage_checkouts,
    file_type,
    voucher_id_array_index,
    expected_status,
    media_root
):
```